test/atat-iam.test.ts

```diff
@@ -23,3 +23,233 @@ describe("ATAT network creation", () => {
+  });
+
+  test("Fully assert generalReadAccess IAM managed policy with matchers", async () => {
+    template.hasResourceProperties(
+      "AWS::IAM::ManagedPolicy",
+      Match.objectEquals({
+        PolicyDocument: {
+          Statement: [
+            {
+              Action: "apigateway:GET",
+              Effect: "Allow",
+              Resource: {
+                "Fn::Join": [
+                  "",
+                  [
+                    "arn:",
+                    {
+                      Ref: "AWS::Partition",
+                    },
+                    ":apigateway:*::/restapis*",
+                  ],
+                ],
+              },
+              Sid: "APIGatewayRestApiReadAccess",
+            },
+            {
+              Action: ["states:List*", "states:Describe*", "states:GetExecutionHistory"],
+              Effect: "Allow",
+              Resource: {
+                "Fn::Join": [
+                  "",
+                  [
+                    "arn:",
+                    {
+                      Ref: "AWS::Partition",
+                    },
+                    ":states:*:",
+                    {
+                      Ref: "AWS::AccountId",
+                    },
+                    ":stateMachine:*",
+                  ],
+                ],
+              },
+              Sid: "StepFunctionsReadAccess",
+            },
+            {
+              Action: ["xray:BatchGetTraces", "xray:Get*", "xray:List*"],
+              Effect: "Allow",
+              Resource: "*",
+              Sid: "XRayReadAccess",
+            },
+          ],
+          Version: "2012-10-17",
+        },
+        Description: "Grants read access to specific resources not in ViewOnlyAccess",
+        Path: "/",
+      })
+    );
+  });
+  test("Fully assert auditorAccess IAM managed policy with matchers", async () => {
+    template.hasResourceProperties(
+      "AWS::IAM::ManagedPolicy",
+      Match.objectEquals({
+        PolicyDocument: {
+          Statement: [
+            {
+              Action: "artifact:Get",
+              Effect: "Allow",
+              Resource: {
+                "Fn::Join": [
+                  "",
+                  [
+                    "arn:",
+                    {
+                      Ref: "AWS::Partition",
+                    },
+                    ":artifact:::report-package/*",
+                  ],
+                ],
+              },
+              Sid: "AllowArtifactReportPackageAccess",
+            },
+            {
+              Action: "artifact:DownloadAgreement",
+              Effect: "Allow",
+              Resource: "*",
+              Sid: "AllowArtifactAgreementDownload",
+            },
+          ],
+          Version: "2012-10-17",
+        },
+        Description: "Grants additional security auditor access beyond SecurityAudit",
+        Path: "/",
+      })
+    );
+  });
+  test("Fully assert developerRwAccess IAM managed policy with matchers", async () => {
```

Comment:
I do think this is a good test overall (mostly because it's something we've broken before). I think the test title isn't ideal though. Maybe:

```
Ensure developers can access CDK-related resources
```
?